lib/utils/dummy_adc.hpp

```diff
@@ -9,7 +9,7 @@ class DummyAdc : public io::IAdc {
   DummyAdc()  = default;
   ~DummyAdc() = default;
 
-  std::optional<std::uint16_t> readValue();
+  virtual std::optional<core::Float> readValue();
```

Comment:
include core types

Reply:
same justification as before, imported in its base class

Reply:
I guess leave it for now and discuss monday?